CircleciScripts/generate_documentation.sh

```diff
@@ -78,7 +78,6 @@ OBJC_SDK_LIST=$(find $SOURCE_ROOT ${SOURCE_ROOT}/AWSAuthSDK/Sources -type d -max
   -not -name "*.xcodeproj" \
   -not -name "*Test*" \
   -not -name "AWSAuthSDK" \
-  -not -name "AWSCognitoIdentityProviderASF" \
```

Comment:
I guess we are generating documentation for this now because it's no longer binary distribution?

Reply:
Correct